.stylelintrc

```diff
@@ -1,3 +1,8 @@
 {
-  "extends": "stylelint-config-bitcrowd"
+  "extends": "stylelint-config-bitcrowd",
+  "rules": {
+    "at-rule-no-unknown": [ true, {
+      "ignoreAtRules": ["else", "include", "mixin", "if", "each", "function", "return", "error", "for", "warn"]
```

Comment:
If we extend our own config in our own lib, should we add this rule to the bitcrowd style config?

Reply:
Possibly 🤔 We generally only need the `include` keyword in other projects (the other stuff is pretty low-level, and only gets used here), but there’s no reason not just add them all tbh — it’s all the sass keywords. I believe there’s also a plugin to add linting all sass keywords.

For now, I’d suggest we leave this here (won’t pass tests otherwise!), make the update over in the stylelint config, then remove this after, wdyt?

Reply:
Sure. It was just a thought when seeing the file 🙂

Reply:
Yup, thanks! I added an issue here: https://github.com/bitcrowd/frontend/issues/24